docs/self-hosted/sso.md

```diff
@@ -115,9 +115,12 @@ REFRESH_TOKEN_COOKIE_SAME_SITE="None"
    - via SDK
 
      ```js
-     const sdk = new Directus('https://directus.myserver.com');
+     import { createDirectus, authentication } from '@directus/sdk';
 
-     await sdk.auth.refresh();
+     const sdk = new Directus('https://directus.myserver.com')
+          .with(authentication('cookie', { credentials: 'include' }));
+
+     await sdk.refresh();
```

Comment:
Cool! Now we should switch to `createDirectus` (plus other suggestion to align with SDK examples) and then we're good to go 🚀 

```suggestion
     const client = new createDirectus('https://directus.example.com')
          .with(authentication('cookie', { credentials: 'include' }));

     await client.refresh();
```